_posts/2021-06-19-release-0.9.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Release of v0.9
+---
+
+gfx-rs community's goal is to make graphics programming in Rust easy, fast, and reliable. Our current main projects are:
+
+  - [gfx-rs](https://github.com/gfx-rs/gfx) makes low-level GPU programming portable with low overhead. It's a single Vulkan-like Rust API with multiple backends that implement it: Direct3D 12/11, Metal, Vulkan, and even OpenGL ES.
+  - [naga](https://github.com/gfx-rs/naga) translates the shaders between languages, including WGSL. Also provides validation and processing facilities on the intermediate representation.
+  - [wgpu](https://github.com/gfx-rs/wgpu) is built on top of gfx-rs and [gpu-alloc](https://github.com/zakarumych/gpu-alloc)/[gpu-descriptor](https://github.com/zakarumych/gpu-descriptor). It provides safety, accessibility, and strong portability of applications.
+
+Following the regular schedule of releasing every few months, we just rolled out the 0.9 version through all of the gfx projects!
+See [gfx-rs changelist](https://github.com/gfx-rs/gfx/blob/master/CHANGELOG.md#hal-090-18-06-2021), [wgpu changelist](https://github.com/gfx-rs/wgpu/blob/master/CHANGELOG.md#v09-2021-06-18), and [naga changelist](https://github.com/gfx-rs/naga/blob/master/CHANGELOG.md#v05-2021-06-18) for the details.
+
+Naga has matured signifgantly since the last release. <naga>
+
+`wgpu` validation has continued continued to improve. Many validation holes were plugged with the last release. Through the combined work in `wgpu` and `naga`, validation holes have been sured up.
+
+`wgpu` perforamnce is still a vital target for us, so we have done work on improving the overhead at resource. <splitting tracker>
+
+## wgpu family reunion and relicense
+
+The biggest internal change since the last release was the progress towards the gfx-rs family reunion. Contributors have long struggled with the amount of effort it takes to add features to wgpu. They would have to make a PR against gfx-hal, then wgpu, then wgpu-rs. 
```

Comment:
here is a question: do you intend to describe changes for 0.9 specifically? this is only moving wgpu-rs in.
Or do you intend to explain the whole set of changes here? In this case, we should make a bigger deal of it than just "biggest change since the last release" :)